mlocal/frags/release_opts.mk

```diff
@@ -1,2 +1,2 @@
-CFLAGS += -O2
+CFLAGS += -O2 -fPIC
```

Comment:
My bad, common_opts.mk is the appropriate location for `-fPIC` flag, so debug/release build have same options